.github/workflows/unit-and-integration-tests-unified.yml

```diff
@@ -17,6 +17,7 @@ name: "Unit & Integration tests CI"
 on:
   push:
     paths-ignore:
+      - '.github/**'
```

Comment:
if the workflow itself changes, wouldn't we want to run those?

Reply:
Removed this ignore rule